pyanaconda/modules/storage/platform.py

```diff
@@ -112,22 +97,51 @@ def stage1_constraints(self):
             PLATFORM_RAID_METADATA: [],
         }
 
+    @property
+    def partitions(self):
+        """The default platform-specific partitions.
+
+        :return: a list of specifications
+        """
+        partitions = [
+            self._bootloader_partition,
+            self._boot_partition
+        ]
+        return list(filter(None, partitions))
+
+    @property
+    def _bootloader_partition(self):
+        """The default bootloader partition for this platform.
+
+        Return the required platform-specific bootloader partition
+        information. These are typically partitions that do not get
+        mounted, like biosboot or prepboot, but may also include
+        the /boot/efi partition.
+
+        :return: a specification or None
+        """
+        return None
+
+    @property
+    def _boot_partition(self):
+        """The default /boot partition for this platform.
+
+        :return: a specification or None
+        """
+        return PartSpec(
+            mountpoint="/boot",
+            size=Size("1GiB")
+        )
+
 
 class X86(Platform):
-    _boot_stage1_device_types = ["disk"]
```

Comment:
I guess this should be removed in previous commit when you were adding ``stage1_constrains``?

Reply:
Fixed.